packages/runtime/client-api/src/api/document.ts

```diff
@@ -215,7 +189,7 @@ function attach(loader: Loader, url: string, deferred: Deferred<Document>): void
             // Check if the Fluid object is viewable
             deferred.resolve(response.value);
         },
-        (error) => {
```

Comment:
Not sure why this code does not propagate errors, but it does not look right. I'd remove it